### PR TITLE
Add shadow dom element

### PR DESCRIPTION
**Pull request by abudevich**

### CODE REVIEW

File: src/main/resources/static/inputs.html [hunk 1]
```diff
@@ -6,6 +6,13 @@
             function handlePress(element) {
                 document.querySelector("#output").innerText = element.value
             }
+
+            function addShadowDomExample(element) {
```
Comment (valfirst) [inline src/main/resources/static/inputs.html:10]:
```suggestion
            function addShadowDom(element) {
```

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — APPROVED** [review]

File: src/main/resources/static/inputs.html [hunk 2]
```diff
@@ -6,6 +6,13 @@
             function handlePress(element) {
                 document.querySelector("#output").innerText = element.value
             }
+
+            function addShadowDom(element) {
```
Comment (ikalinin1) [inline src/main/resources/static/inputs.html:10]: Please move to separate page, there is no reason to have the mixed page for inputs and shadow-dom

Comment (ikalinin1) [reply src/main/resources/static/inputs.html:10]: cc @valfirst

Comment (valfirst) [reply src/main/resources/static/inputs.html:10]: makes sense

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Comment by ikalinin1** [comment]
please rename page shadowDow -> shadowDom

**Review by uarlouski — APPROVED** [review]

File: src/main/resources/static/shadowDow.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Shadow DOM</title>
+    <script>
+            function addShadowDom(element) {
+                const shadow1 = document.querySelector(`#shadow-upper-host`).attachShadow({mode: `open`});
+                shadow1.innerHTML = `<div id="shadow-inner-host"></div>`;
```
Comment (uarlouski) [inline src/main/resources/static/shadowDow.html:8]: why do you need interpolation quotes here?

**Pull request merged**